### PR TITLE
Update material and finish group modals

### PR DESCRIPTION
**Pull request by ranzwertig**
# Definition of Done

- [ ] The code does at least what is defined in the ticket and does not affect any other functionality
- [ ] The PR has a meaningful title
- [ ] There is a link to the issue-id
- [ ] There are no 'hacks' or shortcuts refactoring is preferred
- [ ] The happy case of the app was tested at least once manually

### CODE REVIEW

File: src/app/component/checked/checked.js [hunk 1]
```diff
@@ -6,12 +6,12 @@ import cn from '../../lib/class-names'
 
 import Icon from '../icon'
 
-import checkedIcon from '../../../asset/icon/checked.svg'
-import uncheckedIcon from '../../../asset/icon/unchecked.svg'
+import checkedReoundIcon from '../../../asset/icon/checked-round.svg'
```
Comment (webholics) [inline src/app/component/checked/checked.js]: typo

File: src/app/component/expandable-text/expandable-text.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  const onMoreClick = event => {
+    event.preventDefault()
+    setOpen(!open)
+  }
+
+  return (
+    <span className={cn('ExpandableText', {}, classNames)}>
+      {!open ? (children || '').substring(0, length) : children}{' '}
+      {!open && (
```
Comment (webholics) [inline src/app/component/expandable-text/expandable-text.js]: wir sollten noch prüfen ob der test überhaupt truncated ist, sonst brauchen wir den more link nicht anzeigen

File: src/app/component/expandable-text/expandable-text.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  const [open, setOpen] = useState(false)
+
+  const onMoreClick = event => {
+    event.preventDefault()
+    setOpen(!open)
+  }
+
+  return (
+    <span className={cn('ExpandableText', {}, classNames)}>
+      {!open ? (children || '').substring(0, length) : children}{' '}
```
Comment (webholics) [inline src/app/component/expandable-text/expandable-text.js]: das `|| ''` is unnötig wenn children required ist

File: src/app/component/checked/checked.js (hunk 1 above)
Comment (webholics) [reply src/app/component/checked/checked.js]: still a typo

**Comment by ranzwertig** [comment]
@Dansken I think we need to talk about this

**Pull request merged**